test_debug_cli.py

```
@@ -0,0 +1,129 @@
+import asyncio
+import sys
+import os
+
+# Add current directory to Python path
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+
+async def test_mock_objects():
+    """Test that mock Discord objects work correctly"""
+    print("🧪 Testing mock Discord objects...")
+    
+    from mock_discord import MockUser, MockChannel, MockMessage, MockGuild, create_debug_message
```

Ruff (0.11.9)

`mock_discord.MockGuild` imported but unused

Remove unused import: `mock_discord.MockGuild`

(F401)